e2e/scripts/baseQuery/perf-test.sh

```
@@ -0,0 +1,323 @@
+curl -s -H "X-API-Key: ${API_KEY}" -H "Accept: application/json" \
+  "${API}?${page1_qs}" > "$page1_tmp" 2>/dev/null
+cursor=$(python3 -c "
+import json, sys
+d = json.load(open('$page1_tmp'))
+nc = d.get('nextCursor', '')
+print(nc)
+" 2>/dev/null || echo "")
+rm -f "$page1_tmp"
+
+if [[ -n "$cursor" ]]; then
+  qs=$(build_qs $(langs_params) "size=${PAGE_SIZE}" "cursor=${cursor}")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)